scrapy/pipelines/files.py

```diff
@@ -118,10 +118,10 @@ def _onsuccess(boto_key):
 
         return self._get_boto_key(path).addCallback(_onsuccess)
 
-    def _get_boto_bucket(self):
-        # disable ssl (is_secure=False) because of this python bug:
+    def _get_boto_bucket(self, AWS_SECURE_CONNECTION=False):
+        # adden ability to change secure because of this python bug is fixed on 2.6 and 3.x Python versions:
         # http://bugs.python.org/issue5103
-        c = self.S3Connection(self.AWS_ACCESS_KEY_ID, self.AWS_SECRET_ACCESS_KEY, is_secure=False)
+        c = self.S3Connection(self.AWS_ACCESS_KEY_ID, self.AWS_SECRET_ACCESS_KEY, is_secure=AWS_SECURE_CONNECTION)
         return c.get_bucket(self.bucket, validate=False)
```

Comment:
Just remove the comment lines, if every change to the codebase got commented, it wouldn't be readable anymore.
Move `AWS_SECURE_CONNECTION = False` to Line 85 and change `is_secure=AWS_SECURE_CONNECTION` to `is_secure=self.AWS_SECURE_CONNECTION`.
An argument to `_get_boto_bucket()` wouldn't work.

Also I would prefer it to read `AWS_SECURE_CONNECTION = True`. Security should have priority over questionable backwards compatibility.
thanks